app/models/gamification_leaderboard.rb

```diff
@@ -14,7 +14,8 @@ def self.scores_for(leaderboard_id, page = 0)
 
     join_sql = "LEFT OUTER JOIN gamification_scores ON gamification_scores.user_id = users.id"
     sum_sql = "SUM(COALESCE(gamification_scores.score, 0)) as total_score"
-    users = User.real.joins(join_sql)
+
+    users = User.joins(:user_emails).real.where.not("user_emails.email LIKE '%@anonymized.invalid%'").where(staged: false).joins(join_sql)
```

Comment:
Users and UserEmails tables have a 1 to N relationship, which means that this join *may* multiply the score of a user according to how many emails they got.

Should we pass a `user_emails.primary = true` condition here to avoid this? Or is it handled already by some Rails magic?